docs/en/04_Changelogs/4.0.0.md

```diff
@@ -92,6 +93,138 @@ For a full list of renamed classes, check the `.upgrade.yml` definitions in each
 
 The rename won't affect class-based permission codes or database table names.
 
+##### Use ::class whenever possible
+
+Prefer:
+ ```php
```

Comment:
I avoided that because it's not old/new .. it's a recommended pattern. I didn't want to confuse the reader into thinking one way was deprecated or wrong.